tests/test-min-tls-flag.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+# Creates a ghostunnel. Ensures that /_status endpoint works.
+
+from subprocess import Popen
+from test_common import create_root_cert, create_signed_cert, LOCALHOST, SocketPair, print_ok, cleanup_certs
+import urllib.request, urllib.error, urllib.parse, socket, ssl, time, os, signal, json, sys
+
+if __name__ == "__main__":
+  ghostunnel = None
+  try:
+    # Step 1: create certs
+    create_root_cert('root')
+    create_signed_cert('server', 'root')
+    create_signed_cert('new_server', 'root')
+    create_signed_cert('client1', 'root')
+
+    # Step 2: start ghostunnel, set min TLS version to v1.2
+    ghostunnel = Popen(['../ghostunnel', '--listen={0}:13001'.format(LOCALHOST),
+      '--target={0}:13100'.format(LOCALHOST), '--keystore=server.p12',
+      '--storepass=', '--cacert=root.crt', '--allow-ou=client1',
+      '--status=localhost:13100', '--min-tls=1.2'])
+    time.sleep(5)
```

Comment:
Relying on sleeps are a great way to make sure your tests are too slow or fail on a loaded host